frontend/src/routes/Search.tsx

```diff
@@ -518,25 +542,204 @@ const SearchEvent: React.FC<SearchEventProps> = ({
                         audioOnly,
                     },
                 }}
-                css={{
-                    outline: `1px solid ${COLORS.neutral15}`,
-                    minWidth: 270,
-                    width: 270,
-                    marginLeft: "auto",
-                    [screenWidthAtMost(800)]: {
-                        minWidth: 240,
-                        width: 240,
-                    },
-                    [screenWidthAtMost(BREAKPOINT_MEDIUM)]: {
-                        maxWidth: 400,
-                        margin: "0 auto",
-                    },
-                }}
+                css={thumbnailCss}
             />
         </Item>
     );
 };
 
+const thumbnailCss = {
+    zIndex: 100, // Helps with devTools
```

Comment:
But makes the thumbnails unclickable :/

Reply:
You removed that line but it's still not clickable. The `z-index: 1/2/3` is maybe the problem? Maybe just make the `<a>` element have a higher z index?